scripts/meta/writeBotScripts.py

```diff
@@ -0,0 +1,46 @@
+import os
+from lsst.utils import getPackageDir
+
+isrRunnerScript = """
```

Comment:
if you're going to the trouble of putting a license in your generated script, you should put it in the script generator.

Reply:
lol, indeed